packages/lua-language-server/build.sh

```diff
@@ -0,0 +1,56 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/sumneko/lua-language-server"
+TERMUX_PKG_DESCRIPTION="Sumneko Lua Language Server coded in Lua"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@MrAdityaAlok <dev.aditya.alok@gmail.com>"
+TERMUX_PKG_VERSION=2.3.4
+TERMUX_PKG_SRCURL=https://github.com/sumneko/lua-language-server.git
+TERMUX_PKG_GIT_BRANCH="master"
```

Comment:
The repo seem to get releases fairly often so there does not seem to be a reason to build with git, please change so that it builds from https://github.com/sumneko/lua-language-server/archive/refs/tags/2.3.4.tar.gz instead (we cannot specify version 2.3.4 and build from the git master branch in any case, `master != 2.3.4`)

Reply:
> change so that it builds from https://github.com/sumneko/lua-language-server/archive/refs/tags/2.3.4.tar.gz

How do then I clone submodules? 

I am some what new to `git`....